src/main/java/com/github/tomakehurst/wiremock/core/Options.java

```diff
@@ -136,4 +136,8 @@ default Function<Extensions, NotMatchedRenderer> getNotMatchedRendererFactory()
   Set<String> getTemplatePermittedSystemKeys();
 
   boolean getTemplateEscapingDisabled();
+
+  default boolean failIfMultipleMappingsMatch() {
```

Comment:
This class uses mixed patterns but I guess `getSomething()` dominates. Should it be ` getFailIfMultipleMappingsMatch()` . No strong opinion

Reply:
fixed